crates/ironrdp-pdu/src/rdp/server_license/client_platform_challenge_response.rs

```diff
@@ -179,7 +179,7 @@ enum LicenseDetailLevel {
 }
 
 #[derive(Debug, PartialEq)]
-pub(crate) struct PlatformChallengeResponseData {
+pub struct PlatformChallengeResponseData {
     client_type: ClientType,
     license_detail_level: LicenseDetailLevel,
     challenge: Vec<u8>,
```

Comment:
I’m not sure it’s very useful to export this type with making the fields public too

Reply:
ok